policyengine_us/variables/gov/states/hi/tax/income/exemptions/hi_regular_exemptions.py

```diff
@@ -13,15 +13,12 @@ class hi_regular_exemptions(Variable):
     defined_for = StateCode.HI
 
     def formula(tax_unit, period, parameters):
+        exemptions = tax_unit("exemptions_count", period)
```

Comment:
Use any time this shows up to disambiguate count from amount
```suggestion
        exemptions_count = tax_unit("exemptions_count", period)
```